src/main/kotlin/com/ampnet/userservice/grpc/mailservice/UserDataWithToken.kt

```diff
@@ -0,0 +1,19 @@
+package com.ampnet.userservice.grpc.mailservice
+
+import com.ampnet.userservice.persistence.model.User
+import java.util.UUID
+
+data class UserDataWithToken(
+    val email: String,
+    val coop: String,
+    val language: String,
+    val token: String
+) {
+    constructor(user: User, token: UUID) : this(
+        user.email, user.coop, user.language ?: "", token.toString()
```

Comment:
Why send empty string and let clients (mail-service) decide upon default language. 
I would rather save each user with field language "en" as default value.

Reply:
Good point, but I would rather set empty and leave services to fall back on the default option. This way fall back option will cover both unsupported languages (in the service) and uninitialised language.